packages/blocks/jira-cloud/src/lib/common/index.ts

```diff
@@ -1,25 +1,45 @@
 import {
   AuthenticationType,
+  httpClient,
+  HttpError,
   HttpMessageBody,
   HttpMethod,
   HttpRequest,
   QueryParams,
-  httpClient,
 } from '@openops/blocks-common';
 import { JiraAuth } from '../../auth';
 
 export async function sendJiraRequest(
   request: HttpRequest & { auth: JiraAuth },
 ) {
-  return httpClient.sendRequest({
-    ...request,
-    url: `${request.auth.instanceUrl}/rest/api/3/${request.url}`,
-    authentication: {
-      type: AuthenticationType.BASIC,
-      username: request.auth.email,
-      password: request.auth.apiToken,
-    },
-  });
+  try {
+    return await httpClient.sendRequest({
+      ...request,
+      url: `${request.auth.instanceUrl}/rest/api/3/${request.url}`,
+      authentication: {
+        type: AuthenticationType.BASIC,
+        username: request.auth.email,
+        password: request.auth.apiToken,
+      },
+    });
+  } catch (e) {
+    if (e instanceof HttpError) {
+      const errorBody = e.response?.body || {};
+      const fullError = JSON.stringify(errorBody);
+
+      if (
+        fullError.includes('It is not on the appropriate screen, or unknown.')
+      ) {
+        throw Error(
```

Comment:
the only issue is, as you can see in the screenshot we are basically throwing an error inside another error, idk how much that bothers you or not.

Reply:
So is this why the FE writes "Error 1: Jira, Error 2:" ? 
I think just don't add then `Jira Error:`